openapi.yaml

```diff
@@ -289,7 +289,7 @@ components:
           description: "Root hash of the content"
         originalBytes:
           type: integer
-          format: uint64
+          format: int64
           description: "Length of original content in bytes"
         blockSize:
```

Comment:
This property doesn't have format specified. Should it stay this way?

Reply:
I did this on purpose. I don't think we will ever have a block size larger than int32.max.

Reply:
Ok, but clients consuming API don't know that and they'll not learn it from this spec.